doc/040_cvarlist.md

```diff
@@ -546,6 +546,17 @@ Set `0` by default.
   the controller alone. As these vary by device, it's better to use
   'calibrate' in the 'gamepad' -> 'gyro' menu to set them.
 
+* **joy_haptic_magnitude**: Haptic magnitude value, By default this cvar
+  is `0.0` or disabled. Valid values are 0..1.0.
```

Comment:
A valid value for this cvar can be over 1.0; before I changed the menu option (before rumble), the upper limit was 2.2, and today is 2.0, which of course continues to work for haptic. Maybe don't recommend an upper value? Or have 1.0 as a "recommended default" if you're going to use it?
It's not important, it's just for consistency; we can't tell the user "1.0 is the upper limit" if we offer 2.0/2.2 in the menu.

Reply:
What do you think about updated sentence?

Reply:
I like it!